nine/two_nieoh.py

```diff
@@ -0,0 +1,14 @@
+def symmetric(node):
+
+    def func(node1, node2):
+        if node1 is None and node2 is None:
+            return True
+        if node1 is None or node2 is None:
+            return False
+
+        ret1 = func(node1.left, node2.right)
+        ret2 = func(node1.right, node2.left)
+        ret3 = node1.data == node2.data
+
+        return (ret1 and ret2 and ret3)
```

Comment:
If you just do one monster return statement it might return early in some cases without recursing so much. I think how you have it is much easier to read though :)

Alternatively you could do three `if x: return False` followed by a `return True`